plugins/metas.ts

```diff
@@ -73,7 +73,8 @@ export default function (userOptions?: Partial<Options>) {
       }
 
       const getMetaValue = <T extends keyof MetaData>(key: T) => {
-        if (key in metas) {
+        // null means we should use defaultPageData
+        if (key in metas && metas[key] !== null) {
```

Comment:
I wouldn't be so strict here. Not only `null` but `undefined` or empty strings should be taken into account.
Why not simply `if (!metas[key])` ?

Reply:
Because metas value can be a boolean.

Reply:
Ah, got it!
In this case I would check for `null` and `undefined`.